NeurodataWithoutBorders/pynwb
# docs: Add dynamic versioning string to the sphinx documentation

## Motivation

Fixes #236. Change the hardcoded version strings in the sphinx configuration file to the dynamically created version string by versioneer. Also pep8ified the conf.py file.
## How to test the behavior?
```
pip install -r requirements-doc.txt
cd docs
make html
chromium _build/html/index.html
```
See if the version in the main page is updated. 

## Checklist

- [x] Is your contribution compliant with out coding style ? This can be checked running `flake8` from the source directory.
